components/in_memory_engine/src/region_manager.rs

```diff
@@ -261,6 +286,7 @@ impl CacheRegionMeta {
             in_gc: AtomicBool::new(source_meta.in_gc.load(Ordering::Relaxed)),
             is_written: AtomicBool::new(source_meta.is_written.load(Ordering::Relaxed)),
             evict_info: None,
+            average_cop_requests: Arc::new(Mutex::new(Default::default())),
```

Comment:
So after region split, all child region's stats will be reset. Better derive the stats data from parent region.